src/ring_theory/ideal/operations.lean

```diff
@@ -1714,26 +1634,23 @@ quotient_ker_equiv_of_right_inverse (classical.some_spec hf.has_right_inverse)
 end comm_ring
 
 /-- The kernel of a homomorphism to a domain is a prime ideal. -/
-lemma ker_is_prime {F : Type*} [ring R] [ring S] [is_domain S] [ring_hom_class F R S]
-  (f : F) : (ker f).is_prime :=
+lemma ker_is_prime {F : Type*} [ring R] [ring S] [is_domain S] [ring_hom_class F R S] (f : F) :
+  (ker f).is_prime :=
```

Comment:
On the other hand, I think this is a positive change, since it gives more space to the conclusion.